com.one_ware.OneWare.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.one_ware.OneWare
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: oneware
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --persist=.local/share/OneWareStudio
```

Comment:
persist does not work here since you used host Did you see the line here? https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

> The --persist=DIR option can be used to map directories from the user’s home directory into the sandbox filesystem. This only works if the application has no home or a broader permission like host that includes home.

Reply:
> persist does not work here since you used host Did you see the line here? https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access
> 
> > The --persist=DIR option can be used to map directories from the user’s home directory into the sandbox filesystem. This only works if the application has no home or a broader permission like host that includes home.

Understood, So I have two options: Remove the persist and keep it on host (like other IDEs) or try to make it work with XDG directories. Since this is a code editor that can be used for every kind of code, I prefer the first option.